libs/handlers/handlers.go

```diff
@@ -100,18 +100,6 @@ func WrapValidationError(err error) *AppError {
 	return ValidationError("request body", govalidator.ErrorsByField(err))
 }
 
-// CodedValidationError creates an error to communicate a bad request was formed
-func CodedValidationError(message string, errorCode string, validationErrors interface{}) *AppError {
-	return &AppError{
-		Message:   "Error validating " + message,
-		ErrorCode: errorCode,
-		Code:      http.StatusBadRequest,
-		Data: map[string]interface{}{
-			"validationErrors": validationErrors,
-		},
-	}
-}
-
```

Comment:
This code did not deserve its place in `lib`, as it was only used in one place. Additionally, it has led to poorly formatted error messages (since `message` would often contain also `Error doing something` which resulted in error messages such as `Error validating Error doing something`).